test/testcafe/spec/ChallengeAuthenticatorEmail_spec.js

```diff
@@ -46,31 +46,42 @@ const magicLinkTransfer = RequestMock()
   .respond(terminalTransferedEmail);
 
 
-fixture(`Challenge Email Authenticator Form`);
+fixture('Challenge Email Authenticator Form');
 
 async function setup(t) {
   const challengeEmailPageObject = new ChallengeEmailPageObject(t);
-  challengeEmailPageObject.navigateToPage();
+  await challengeEmailPageObject.navigateToPage();
   return challengeEmailPageObject;
 }
 
 test
-  .requestHooks(validOTPmock)(`challenge email authenticator screen has right labels`, async t => {
+  .requestHooks(validOTPmock)('challenge email authenticator screen has right labels', async t => {
     const challengeEmailPageObject = await setup(t);
+
+    const { log } = await t.getBrowserConsoleMessages();
+    await t.expect(log.length).eql(3);
+    await t.expect(log[0]).eql('===== playground widget ready event received =====');
+    await t.expect(log[1]).eql('===== playground widget afterRender event received =====');
+    await t.expect(JSON.parse(log[2])).eql({
+      controller: 'mfa-verify-passcode',
+      formName: 'challenge-authenticator',
+      authenticatorType: 'email',
+    });
+
     const pageTitle = challengeEmailPageObject.getPageTitle();
     const saveBtnText = challengeEmailPageObject.getSaveButtonLabel();
     await t.expect(saveBtnText).contains('Verify');
     await t.expect(pageTitle).contains('Verify with your email');
     await t.expect(challengeEmailPageObject.getFormSubtitle())
-      .contains(`An email was sent to`);
+      .contains('An email was sent to');
```

Comment:
curious why we are doing this change ?

Reply:
It's unnecessary to use string literal but just string. okta-ui has same rule.